src/integrations/prefect-gcp/prefect_gcp/workers/vertex.py

```diff
@@ -184,6 +183,26 @@ class VertexAIWorkerVariables(BaseVariables):
             "and required if a service account cannot be detected in GCP credentials."
         ),
     )
+    enable_web_access: bool = Field(
+        default=False,
+        title="Enable Web Access",
+        description=(
+            "Whether you want Vertex AI to enable `interactive shell access` "
+            "See https://cloud.google.com/vertex-ai/docs/training/monitor-debug-interactive-shell for how to access your job via interactive console when running."
+        ),
+        examples=[True],
+    )
+    enable_dashboard_access: bool = Field(
+        default=False,
+        title="Enable Dashboard Access",
+        description=(
+            "Whether you want Vertex AI to enable access to the customized dashboard in the training chief container. If set to true, you can access the dashboard "
+            "in the Cloud Console UI. Inspect the Vertex AI in the Cloud Console UI, go to Training and then Custom Jobs. Click on the name of your "
+            "custom training job. On the page for your job, click `Launch web terminal` for `workerpool0-0:` + `port number` for dashboard access. "
+            "See https://cloud.google.com/vertex-ai/docs/training/monitor-debug-interactive-shell#get-uri"
```

Comment:
I think keeping this pithy and linking to the docs will look better in the UI
```suggestion
            "Whether to enable access to the customized dashboard in the training chief container. " 
            "See https://cloud.google.com/vertex-ai/docs/training/monitor-debug-interactive-shell#get-uri for access instructions."
```